heltour/tournament/views.py

```diff
@@ -24,6 +24,7 @@
 from django.views.generic import View
 from icalendar import Calendar, Event
 
+from heltour.settings import HELTOUR_VERSION
```

Comment:
Use django.conf settings
```suggestion
from django.conf import HELTOUR_VERSION
```

Reply:
https://docs.djangoproject.com/en/5.2/topics/settings/#using-settings-in-python-code